### ISSUE TITLE
Add 'Updated' timestamp

### ISSUE THREAD

**Issue by BogdanDogaru** [bbc/simorgh#606]
**Is your feature request related to a problem? Please describe.**
We need to display the absolute and relative timestamp to indicate when the article was last updated.

**Describe the solution you'd like**
Using the investigation in #604 and any following discussions, implement the chosen solution to show the `lastUpdated` date in a user-friendly format, according to the table below:

| relative timestamp | last updated message| 
| ---------- | -------- |
| < 2 min ago | Updated 1 minute ago| 
| 2 min ago <= timestamp < 1 hr ago | Updated `M` minutes ago| 
| 1 hr ago <= timestamp M < 2 hr ago | Updated 1 hour ago| 
| 2 hr ago <= timestamp < 10 hours ago | Updated `H` hours ago| 
| First published today && updated timestamp >= 10 hours ago | Updated `HH:MM XXT` i.e. 10:52 GMT| 
| First published not today && updated timestamp >= 10 hours ago | Updated `DD Mmm YYYY`| 

UX details:
- font family: Reith Sans Regular
- size (type): brevier
- colour: `#5a5a5a` (rhino)
- padding-top: 0px
- padding-bottom: 16px
- a11y
    - HTML: no meaning e.g. span, except for time element, example markup 
`<span class="timestamp__updated">Updated <time datetime="2018-07-13">58 minutes ago</time></span>`
    - sits within the "main" region

Notes:
- there should be no visual ordinal indicator (i.e. "st/nd/rd/th")
- only show this if the last updated time > first published time
- the "Updated" text should be configurable, for easier integration with a future configuration/localisation service

For more details about edge cases see https://confluence.dev.bbc.co.uk/display/NEWSART/Article+-+Timestamps

**Testing notes**
TBD

**Comment by BogdanDogaru** [comment]
Blocked on:
- [ ] Investigation in #604

**Comment by jimjohnsonrollings** [comment]
Blocked on:

- [ ] Jim updating the requirements

**Comment by bcmn** [comment]
Closing in favour of this sequence of tickets:
#1277
#1282
#1279
#1280
#1283
#1284
